src/components/App.css

```diff
@@ -1,22 +1,39 @@
-.App {
-  text-align: center;
+.app{
+  width: 700px;
+  margin: 0 auto;
 }
 
-.App-logo {
-  height: 40vmin;
+.display{
+  background: grey;
+  max-height: 100px;
+  line-height: 100px;
+  color: #fff;
+  font-weight: bold;
+  font-size: 2em;
+  text-align: right;
+  padding: 2em;
 }
 
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
+.calc-btn{
+  height: 100px;
+  font-weight: bold;
+  border: 0.0015em solid grey;
+  width: 25%;
+}
+
+.button-panel{
   display: flex;
   flex-direction: column;
-  align-items: center;
-  justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
+}
```

Comment:
`App`, `Display` and `ButtonPanel` components should use flex display mode
You only did it for `ButtonPanel`
https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/useFlex.png